src/app/components/Login.tsx

```diff
@@ -0,0 +1,8 @@
+"use client"
+import { Button } from "@mui/material";
+import { signIn } from "next-auth/react";
+export default function Login() {
+  return <Button onClick={() => signIn("keycloak")} variant="contained" color="secondary">
+    Sign in
+  </Button>
+}
```

Comment:
Similar comment around adding in some line separators. This is just to make the code a bit more readable